src/components/Dropdown/Dropdown.tsx

```diff
@@ -88,22 +90,23 @@ const DropdownButton = styled.button`
     border-radius: 6px;
     background-color: unset;
     cursor: pointer;
-    :focus-visible {
-      box-shadow: 0 0 0 2px ${transparentize(0.6, theme.colors.black[200])};
+    &:focus-visible {
+      outline: 1px solid transparent;
+      box-shadow: 0 0 0 2px ${theme.colors.black[200]};
     }
   `}
 `
 
 export const Dropdown = ({ children, trigger }: DropdownProps) => {
```

Comment:
Sugestao de melhoria:
```typescript
 <Dropdown>
          <DropdownTrigger>
            <ProfileName size={2} name={maybeData.value.user.username} />
            </DropdownTrigger>
          <DropdownContent>
            <DropdownList>
            <DropdownItem
            href={`/profile/${maybeData.value.user.username}`}
            label="Profile"
          />
          <DropdownItem label="Sign Out" onEventClick={signOut} />
            </DropdownList>
          </DropdownContent>
        </Dropdown>
```

Reply:
What do you think about this [way](https://github.com/joaooliveiradev/conduit-frontend/pull/43/commits/fb2da6a52a2b63069bfafd44df306a5995fcad8d)?